tyk-headless/templates/service-gw-control.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.gateway.control.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
```

Comment:
Could you please add the `annotations` too, like on this service?
https://github.com/TykTechnologies/tyk-helm-chart/blob/12213293cf1eaa1f23edfe638167e4181c85c10d/tyk-headless/templates/service-gw.yaml#L10-L15

Reply:
added  gateway control annotations